src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1136,11 +1142,7 @@ void NetworkOPsImp::apply (std::unique_lock<std::mutex>& batchLock)
             }
             else if (isTerRetry (e.result))
             {
-                if (e.failType == FailHard::yes)
-                {
-                    addLocal = false;
-                }
-                else
+                if (e.failType != FailHard::yes)
```

Comment:
seems like `addLocal` is a direct alias for `e.local` now (never modified), so we can just get rid of it.

Reply:
Sounds good, done.